examples/Python/snowboythreaded.py

```diff
@@ -0,0 +1,94 @@
+import snowboydecoder
+import threading
+import Queue
+
+
+class ThreadedDetector(threading.Thread):
+    """
+    Wrapper class around detectors to run them in a separate thread
+    and provide methods to pause, resume, and modify detection
+    """
+
+    def __init__(self, models, **kwargs):
+        """
+        Initialize Detectors object. **kwargs is for any __init__ keyword arguments to be passed into HotWordDetector
```

Comment:
Make sure you don't put more than 80 characters in a line. Same for other places.